### PR TITLE
Nav

### PR DESCRIPTION
**Pull request by YakoobHammouri**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,228 +1,57 @@
...
-     margin: 30px 30px 30px;
-     padding: 20px 20px 20px; 
-    border: #00bfd8;
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-   
-    
+html {
```
Comment (bayan-404) [inline style.css:13]: is that a class or id ? 
yup you need to reExplain it

File: style.css [hunk 2]
```diff
@@ -1,228 +1,57 @@
...
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    margin-top:10px;
-    margin-bottom:5px;
-    width: 520px;
-    
- }
-
- .userInfo > input{
```
Comment (bayan-404) [inline style.css:43]: does it work ? class = "userInfo > input"?

File: style.css [hunk 3]
```diff
@@ -281,18 +123,14 @@ a {
...
 /*============================ navgition ======================================*/
 
 .navgition {
     width: 100%;
     padding: 10px 0;
-    -webkit-transition: all 0.3s ease-out 0s;
-    -moz-transition: all 0.3s ease-out 0s;
-    -ms-transition: all 0.3s ease-out 0s;
-    -o-transition: all 0.3s ease-out 0s;
-    transition: all 0.3s ease-out 0s;
```
Comment (bayan-404) [inline style.css:295]: ?

File: style.css [hunk 4]
```diff
@@ -305,70 +143,54 @@ a {
...
-    font-size: 18px;
-    line-height: 35px;
-    font-weight: 700;
-    padding: 26px 0;
-    color: #121212;
-    text-transform: uppercase;
-    position: relative;
-    transition: all 0.3s ease-out 0s;
-}
+    .navgition__nav ul li a {
```
Comment (bayan-404) [inline style.css:151]: :0

File: style.css [hunk 5]
```diff
@@ -305,70 +143,54 @@ a {
...
+        line-height: 35px;
+        font-weight: 700;
+        padding: 26px 0;
+        color: #121212;
+        text-transform: uppercase;
+        position: relative;
+        transition: all 0.3s ease-out 0s;
+    }
 
-.navgition__nav ul li>a::before {
```
Comment (bayan-404) [inline style.css:335]: :0 ?

File: style.css [hunk 6]
```diff
@@ -378,7 +200,7 @@ a {
     margin: 0 15px 0 0;
 }
 
-.navgition__followUs ul li>a {
```
Comment (bayan-404) [inline style.css:381]: delete this :)

File: style.css [hunk 7]
```diff
@@ -393,23 +215,185 @@ a {
     list-style: none;
 }
 
-.navgition ul li {
```
Comment (bayan-404) [inline style.css:396]: delete this too :)

File: style.css [hunk 8]
```diff
@@ -393,23 +215,185 @@ a {
...
+.home .p-large {
+    margin-bottom: 24px;
+    color: #626262;
+}
+/*============================================ home  =============================================*/
+
+
+
+
+/*============================= about us , Bayan ===================*/
```
Comment (bayan-404) [inline style.css:258]: repeated

File: style.css [hunk 9]
```diff
@@ -393,23 +215,185 @@ a {
...
+    box-sizing: border-box;
+    flex-basis: 31em;
+}
 
-/*================================================ roba ==================================*/
+/*============================= about us , Bayan ===================*/
+
+
+
+/*===================Contact , Hnan ====================*/
```
Comment (bayan-404) [inline style.css:303]: repeated

File: style.css [hunk 10]
```diff
@@ -483,201 +472,337 @@ a {
...
+        max-width: 720px;
     }
-    /* Modile */
+
+    .navgition__followUs {
+        display: flex !important;
+    }
+
+
+    .con {
```
Comment (bayan-404) [inline style.css:686]: repeated

File: style.css [hunk 11]
```diff
@@ -483,201 +472,337 @@ a {
...
+    .con__par {
+        flex-basis: 6em;
+        padding-left: 10px;
+        padding-right: 10px;
+    }
+}
+
+
+
+@media(min-width :992px) and (max-width:1199) {
```
Comment (bayan-404) [inline style.css:705]: repeated

**Review by bayan-404 — APPROVED** [review]

**Review by Hanan795 — APPROVED** [review]

**Review by rubasider — APPROVED** [review]

**Review by rubasider — APPROVED** [review]

**Review by Hanan795 — APPROVED** [review]

**Pull request merged**